### PR TITLE
Add homework jdi

### PR DESCRIPTION
**Pull request by serpencrow**

### CODE REVIEW

File: src/main/java/hw_jdi/Utils.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+        if (isNotEmpty(formData.getElements())) {
+            StringBuilder builder = new StringBuilder("Elements: ");
+            AtomicInteger elementsCount = new AtomicInteger(formData.getElements().size());
+
+            for (String element : formData.getElements()) {
+                builder.append(element);
+
+                if (elementsCount.get() != 1) {
+                    builder.append(", ");
```
Comment (troe88) [inline src/main/java/hw_jdi/Utils.java]: Take a look at String.join

File: src/main/java/hw_jdi/entities/SummaryForm.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package hw_jdi.entities;
+
+public class SummaryForm {
```
Comment (troe88) [inline src/main/java/hw_jdi/entities/SummaryForm.java]: What is the reason of this class ?

File: src/main/java/hw_jdi/entities/MetalsAndColorsForm.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    @XPath("//div[@id='colors']//button")
+    private Button colorDropdownExpander;
+
+    @XPath("//div[@id='metals']//button")
+    private Button metalsDropdownExpander;
+
+    @XPath("//div[@id='vegetables']//button")
+    private Button vegetablesDropdownExpander;
+
+    public void fillForm(final MetalsAndColorsFormData metalsAndColorsFormData) {
```
Comment (troe88) [inline src/main/java/hw_jdi/entities/MetalsAndColorsForm.java]: Why don't you override fill(Entity ...)  method ?

Comment (serpencrow) [reply src/main/java/hw_jdi/entities/MetalsAndColorsForm.java]: Fixed

File: src/main/java/hw_jdi/entities/SummaryForm.java (hunk 2 above)
Comment (serpencrow) [reply src/main/java/hw_jdi/entities/SummaryForm.java]: Originally this class was supposed to be used inside the MetalsAndColorsForm form, but was not needed. Deleted

File: src/main/java/hw_jdi/Utils.java (hunk 1 above)
Comment (serpencrow) [reply src/main/java/hw_jdi/Utils.java]: Replaced StringBuilder to String.join()

File: src/main/java/hw_jdi/entities/SummaryForm.java (hunk 2 above)
Comment (serpencrow) [reply src/main/java/hw_jdi/entities/SummaryForm.java]: Originally this class was supposed to be used inside the MetalsAndColorsForm form, but was not needed. Deleted

**Pull request merged**